src/components/Dialog/Dialog.st.css

```diff
@@ -39,7 +44,23 @@
     left: 50%;
     transform: translate(-50%, -30%);
 
-    background: #fff;
+    background: value(DefaultBackgroundColor);
+}
+
+.closeButtonWrapper {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+}
+
+.closeIconButton {
+    -st-mixin: TPAIconButton(
```

Comment:
don't you want to use here the `overrideStyleParams` pattern?

Reply:
I thought about it, but we'll need to support both approaches
for backwards compatibility.
Or we can break the api, and talk to whoever's using it